android/travis_build_all.sh

```diff
@@ -12,4 +12,19 @@ set -e
 ./buildAndroidBOINC-CI.sh --cache_dir "$ANDROID_TC" --build_dir "$BUILD_DIR" --silent --ci --arch x86
 ./buildAndroidBOINC-CI.sh --cache_dir "$ANDROID_TC" --build_dir "$BUILD_DIR" --silent --ci --arch x86_64
 
+if [ ! -d ../3rdParty/buildCache/android-tc/arm ]; then
+    echo You run this script from diffrent folder: $PWD
+    echo Please run it from boinc/android
+    exit 1
+fi
+
+list_libs="libcrypto.a libssl.a libcurl.a"
+
+for i in $list_libs; do
```

Comment:
Since this particular piece of code is needed in two separate scripts, maybe it's reasonable to move it to `android/buildAndroidBOINC-CI.sh`?